var/spack/repos/builtin/packages/acts/package.py

```diff
@@ -224,6 +225,7 @@ class Acts(CMakePackage, CudaPackage):
     depends_on("boost @1.71: +filesystem +program_options +test", when="@0.10.4:")
     depends_on("cmake @3.14:", type="build")
     depends_on("dd4hep @1.11: +dddetectors +ddrec", when="+dd4hep")
+    depends_on("dd4hep @1.21: +dddetectors +ddrec", when="@20: +dd4hep")
```

Comment:
Does this also apply to m`main`?

Reply:
Yes, spack has a few special "infinity" versions like `main`, `master`, `develop` that are treated as newer than any numeric versions in comparisons